scripts/vagrant/provision/setup_privileged.sh

```diff
@@ -3,15 +3,33 @@
 # Use with Ubuntu 16.x+
 set -xe
 
+apt-get update
+
+DOCKER_CONFIG_FILE=daemon.json
+
+# Mount nvme devices (if present)
+apt-get install -y mhddfs
+DISKS=$(lsblk | grep nvme | awk '{ print $1 }')
```

Comment:
Would it be slightly more portable if we made this `grep -i nvme` in case another provider names it with capitals somewhere in the disk name?